lib/v2/qingting/podcast.js

```diff
@@ -0,0 +1,75 @@
+const crypto = require('crypto');
+const got = require('@/utils/got');
+
+module.exports = async (ctx) => {
+    const channelUrl = `https://i.qingting.fm/capi/v3/channel/${ctx.params.id}`;
+    let response = await got({
+        method: 'get',
+        url: channelUrl,
+        headers: {
+            Referer: 'https://www.qingting.fm/',
+        },
+    });
+    const title = response.data.data.title;
+    const channel_img = response.data.data.thumbs['400_thumb']
+    const authors = response.data.data.podcasters.map((author) => author.nick_name).join(',');
+    const desc = response.data.data.description;
+    const programUrl = `https://i.qingting.fm/capi/channel/${ctx.params.id}/programs/${response.data.data.v}?curpage=1&pagesize=10&order=asc`;
+    response = await got({
+        method: 'get',
+        url: programUrl,
+        headers: {
+            Referer: 'https://www.qingting.fm/',
+        },
+    });
+
+    const resultItems = await Promise.all(
+        response.data.data.programs.map((item) =>
+            ctx.cache.tryGet("" + item.id, async () => {
```

Comment:
Maybe use the `link` below as the cache key as it has a lower chance of collision